lib/mrtrix3/dwi2mask/synthstrip.py

```diff
@@ -0,0 +1,114 @@
+# Copyright (c) 2008-2023 the MRtrix3 contributors.
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+# Covered Software is provided under this License on an "as is"
+# basis, without warranty of any kind, either expressed, implied, or
+# statutory, including, without limitation, warranties that the
+# Covered Software is free of defects, merchantable, fit for a
+# particular purpose or non-infringing.
+# See the Mozilla Public License v. 2.0 for more details.
+#
+# For more details, see http://www.mrtrix.org/.
+
+import shutil
+import argparse
+import os
+import warnings
+from mrtrix3 import MRtrixError
+from mrtrix3 import app, run
+
+
+DEFAULT_CLEAN_SCALE = 2
+SYNTHSTRIP='mri_synthstrip'
+
+
+def usage(base_parser, subparsers): #pylint: disable=unused-variable
+  parser = subparsers.add_parser('synthstrip', parents=[base_parser])
+  parser.set_author('Ruobing Chen(chrc@student.unimelb.edu.au)')
+  parser.set_synopsis('Use the Synthstrip heuristic (based on skull-stripping method)')
+  parser.add_argument('input',  help='The input DWI series')
+  parser.add_argument('output', help='The output mask image')
+  options=parser.add_argument_group('Options specific to the \' Synthstrip \'algorithm')
+  options.add_argument('-s', action='store_true', default=False, help='The output stripped image')
```

Comment:
Rather than a "`store_true`" action (in MRtrix3 parlance we'd typically call this a command-line option "flag"), where it's just the "`-s`" that gets specified at the command-line, this should instead just be a "standard" command-line option (ie. no `action` or `default` kwargs). The user can then provide at the command-line, after the "`-s`", the location at which they wish for the (optional) stripped image to be saved.